### PR TITLE
Fix for small typos on documentation 

### PR DESCRIPTION
**Pull request by beyildirim**
I have fixed some typos in the gettingstarted, vagrant, and installation sections of the documentation.

If there are any mistakes with the PR(commits, description, etc.) please let me know.

### CODE REVIEW

File: Documentation/gettingstarted/hubble.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Enable the Hubble UI by running the following command:
             cilium hubble enable --ui
             🔑 Found existing CA in secret cilium-ca
             ✨ Patching ConfigMap cilium-config to enable Hubble...
-            ♻️  Restarted Cilium pods
+            ♻️ Restarted Cilium pods
```
Comment (qmonnet) [inline Documentation/gettingstarted/hubble.rst:43]: Is this one simply a typo in the docs, or does Hubble mistakenly add this additional space? Would be nice to fix Hubble as well, if it's responsible for the typo.

**Review by qmonnet — APPROVED** [review]
Looks good, thank you!

File: Documentation/gettingstarted/hubble.rst (hunk 1 above)
Comment (beyildirim) [reply Documentation/gettingstarted/hubble.rst:43]: It was coming from cilium-cli, detected and fixed that typo as well: https://github.com/cilium/cilium-cli/pull/2690

**Review by rolinh — APPROVED** [review]

**Comment by joestringer** [comment]
Hi @beyildirim, thanks for the contributions. A couple of tips:
- Rather than merging the upstream branch into your development branch, please use `git pull --rebase` to rebase your changes on top of the latest upstream changes. This avoids the merge commits which are also not signed off, so it will resolve the issues with the bot requesting changes to your PR.
- Unless GitHub reports that there is a specific conflict in some file, there is generally no need to update the branch vs. the `main` branch. GitHub will often describe that "This branch is out-of-date with the base branch". However, changes do not need to be up-to-date. Cilium merges new changes into the tree very frequently, so the odds are that the PR is out-of-date very quickly after you update it.

Could you rebase the PR to remove the 'merge' commits and force push the branch into your repo to update this PR? Thanks.

**Review by joestringer — APPROVED** [review]

**Pull request merged**